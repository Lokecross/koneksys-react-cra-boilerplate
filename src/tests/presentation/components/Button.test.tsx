import React from 'react';
import { shallow } from 'enzyme';
import { Button } from '@components';

test('Renders Button component and match to previous snapshot', () => {
  const component = shallow(<Button />);
  expect(component).toMatchSnapshot();
});
