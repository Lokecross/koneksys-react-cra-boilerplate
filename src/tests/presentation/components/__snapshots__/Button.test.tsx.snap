// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Button component and match to previous snapshot 1`] = `<styled.button />`;
